fv3core/examples/standalone/build_scripts/build_example.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# Example bash script to build Pace to run locally outside of a container, assuming Pace is cloned into $HOME
+
+set -e -x
+
+# module load necessary system software
+module load gcc/11.3.0
+module load openmpi
+module load cuda/10.2
+module load python/3.8.13
+
+# download boost and add to path:
+mkdir src
+cd src
+wget https://boostorg.jfrog.io/artifactory/main/release/1.79.0/source/boost_1_79_0.tar.gz
+tar -xzf boost_1_79_0.tar.gz
+mkdir -p boost_1_79_0/include
+mv boost_1_79_0/boost boost_1_79_0/include/.
+setenv BOOST_ROOT $HOME/src/boost_1_79_0
+
+# go into pace, update submodules and download test data:
+cd $HOME/pace
+git submodule update --init
+cd fv3core
+make get_test_data
+cd ..
```

Comment:
Scripts should not change the CWD.

Reply:
Also this assumes `pace` is located at `$HOME/pace`, which might not be the case (if building in `scratch`).